src/mergeSort.js

```diff
@@ -0,0 +1,30 @@
+const mergeSort = (inputArray) => {
+  const middle = inputArray.length / 2
+  const left = inputArray.slice(0, middle)
+  const right = inputArray.slice(middle, inputArray.length)
+  if( inputArray.length < 2) {
+    return inputArray
+  }
+  return merge(mergeSort(left), mergeSort(right))
+
+}
+
+const merge = (left, right) => {
+  const finalArray = []
+  while(left.length || right.length){
+    if(left.length && right.length){
```

Comment:
I think using boolish evaluation is confusing so I dont like it.
I would explicitly state what I am comparing each value to.
Pero laik, dats jus me.